### PR TITLE
Bug 1184476 - Handle NotFoundError from DOMError r=daleharvey

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=415ecf5c50bb7f5ce3c882aba86aad489d305523

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=51e582b7a6d86bd0c4ea85bed6066a47454e48e2

File: apps/system/locales/system.en-US.properties [hunk 1]
```diff
@@ -250,10 +250,12 @@ screenshotSDCardLow   = Not enough space on memory card
 logsSavedHeader = Device logs saved
 logsSavedBody = Tap to share logs
 logsSaveError = Error while saving logs
+logsShareError = Error while sharing logs
 logsSaving = Capturing logs…
 logsOperationFailed = Operation {{operation}} failed.
 logsGenericError = Unexpected error occurred.
 logsSDCardMaybeShared = Make sure your SD card is not shared with your computer.
+logsNotFoundError = At least one log file could not read from the filesystem.
```
Comment (JohanLorenzo) [inline apps/system/locales/system.en-US.properties]: Nit: `could not be read `

File: apps/system/js/devtools/logshake.js [hunk 2]
```diff
@@ -196,12 +202,16 @@
...
+        if ('name' in error && error.name === 'NotFoundError') {
+          return navigator.mozL10n.get('logsNotFoundError');
+        }
       }
 
       return '';
     },
 
-    showErrorMessage: function(error, notif) {
+    showErrorMessage: function(title, error, notif) {
```
Comment (JohanLorenzo) [inline apps/system/js/devtools/logshake.js]: Is it possible to protect the user from displaying multiple times the error message? Example: If you triple tap the notification, you currently have to dismiss 3 error messages.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=773baea1b6961fc8660683837d73781f8daefc27

File: apps/system/js/devtools/logshake.js (hunk 2 above)
Comment (lissyx) [inline apps/system/js/devtools/logshake.js]: I don't think it's big deal for now. Might make the code more complex for an edge case.

**Pull request merged**